src/inc/quic_hashtable.h

```diff
@@ -246,6 +246,9 @@ QuicHashTableContract(
 // Simple helper hash function.
 //
 inline
+#if defined(__clang__)
+    __attribute__((no_sanitize("unsigned-integer-overflow")))
+#endif
```

Comment:
I don't like #ifdef (some platform specific thing) in common code. I prefer some platform abstraction layer, that is no-op on platforms it doesn't apply to. For instance:
```C
#if defined(__clang__)
#define QUIC_NO_SANITIZE(X) __attribute__((no_sanitize(X)))
#else
#define QUIC_NO_SANITIZE(X) 
#endif
```

Reply:
The problem is that "X" is compiler-specific.

Reply:
Which file should the `QUIC_NO_SANITIZE` define go into?

Reply:
It would go in each individual platform header.